components/Projects.vue

```diff
@@ -27,6 +27,17 @@ export default {
       required: true,
     },
   },
+  computed: {
+    projectsSortedByName() {
+      return [...this.content].sort((a, b) =>
+        a.name.toLowerCase() > b.name.toLowerCase()
+          ? 1
+          : a.name.toLowerCase() < b.name.toLowerCase()
+          ? -1
+          : 0
```

Comment:
There is a function called `localeCompare` that returns exactly the thing you need! 😄 
You can refactor the previous code like this:

```js
a.name.toLowerCase().localeCompare(b.name.toLowerCase())
```

More info here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare

It has great support as well https://caniuse.com/?search=localeCompare

Reply:
I didn't know about it, thanks a lot! 👏🏻